client/utils/charge/index.ts

```diff
@@ -5,39 +5,51 @@
  */
 import { sumBy } from 'lodash';
 
+/**
+ * Internal dependencies
+ */
+import { Dispute } from 'types/disputes';
+import { Charge, ChargeAmounts } from 'types/charges';
+
 const failedOutcomeTypes = [ 'issuer_declined', 'invalid' ];
 const blockedOutcomeTypes = [ 'blocked' ];
 
-export const getDisputeStatus = ( dispute = {} ) => dispute.status || null;
+export const getDisputeStatus = ( dispute: Dispute = {} as Dispute ): string =>
```

Comment:
is `Dispute = {} as Dispute` a way of dealing with cases when an empty object is returned for non existing dispute?

Reply:
I found it as a way to keep linter happy, but I myself is not so happy about it, as resulting object won't contain all the required properties. @kalessil, maybe you know a better approach for this case?

Reply:
Haven't tested but could we not define a type 
```
interface FullDispute {...}
type Dispute = FullDispute | Record< string: never >
```

Reply:
Looking into @dmvrtx

Reply:
@dmvrtx : similar to youth, but alternative syntax `dispute: Dispute = <Dispute>{}`. Looks pretty much as type casting.

Reply:
I like the brevity of it. Will update PR later.

Reply:
Done in 0d76ca92